machAeroTutorials/overset_volume_mesh.rst

```diff
@@ -0,0 +1,187 @@
+``near_tip`` mesh. This individual options overwrite the common options if the same key exists in both of them.
+
+inputFile
+    Since we have different surface meshes, we have to supply the inputfile name individually
+outputFile
+    We also want different output names. This way we can inspect the generated mesh separately
+BC 
+    Here we apply the boundary conditions (BC). The integer defines the Domain (starting at 1). The dict key defines
+    which side of the domain the BC applies to. The dict value defines the type of the BC. 
+
+    As it has been mentioned in the previous tutorial, there is not a reliable way to get this integer, which defines
+    the domain, out of Pointwise. So it is recommended to rotate all domains in such a way, that the BC can be applied on 
+    the same side of all domains. Then they are deleted one by one until no 
+    more error messages pop up in pyHyp.
+families
+    Here we give a unique name to a surface. This lets ADflow calculate the forces seperately and would allow you, for example,
+    to get the lift and drag forces for your wing and tail individually
+
+
+Extrude the nearfield
+---------------------
+
+Now we extrude the nearfield:
+
+.. literalinclude:: ../tutorial/overset/mesh/run_pyhyp.py
+   :start-after: # rst near_field (beg)
+   :end-before: # rst near_field (end)
+
+We start the extrusion by calling ``pyHypMulti``\. As arguments we give the previously defined common and individual options.
+After the extrusion, we wait for all procs to finish before we continue.
+
+
+Combine the nearfield
+---------------------
+The farfield consist of a cartesian part in the middle and a simple Ogrid around it. This cartesian part will 
+enclose all the nearfields. Because of that, we have to combine all the nearfields first:
+
+.. literalinclude:: ../tutorial/overset/mesh/run_pyhyp.py
+   :start-after: # rst combine_near_field (beg)
+   :end-before: # rst combine_near_field (end)
+
+First, the script reads the files, then it combines them. In the end everything gets moved to ``y=0``.
+
+
+Generate the farfield
+---------------------
+Now we can generate the farfield:
+
+.. literalinclude:: ../tutorial/overset/mesh/run_pyhyp.py
+   :start-after: # rst far_field (beg)
+   :end-before: # rst far_field (end)
+
+These are the arguments for a ``simpleOCart``:
+
+dhStar
+    The spacing in the cartesian part
+40\.
+    The extend of the farfield. Measured in the longest diagonal lenght in the cartesian part
+nFarfield
+    How many layers the Ogrid part should have
+'y'
+    On which axis the symmetry lies
+1
+    How many MultiGrid-Cicles are enforced during the generation
+farfield
+    The outputfile name for the farfield
+
+Combine everything
+------------------
+Here we combine all the meshes into one. We do this only on the root processor if we run it in parallel.
+
+.. literalinclude:: ../tutorial/overset/mesh/run_pyhyp.py
+   :start-after: # rst combine (beg)
+   :end-before: # rst combine (end)
+
```

Comment:
It would be great to have a picture here of the final mesh

Reply:
@joanibal I see that this comment is still not addressed. Do you want to have this figure? It is kinda difficult to show a figure of the overset volume mesh so I am not sure how it would look like. Let me know if you still want this figure.